### ISSUE TITLE
Doesn't need to refreshes too often.

### ISSUE THREAD

**Issue by huanfe1** [RSSNext/Follow#1355]
### Describe the bug

There are too many refreshes! RSS user don't need much real-time.

When switch category in sidebar, doesn't need to load too often, can use the existing timing to load and click header refetch button.

Too many switch also cause the user's avatar to reload.

This is good for both the user experience and the follow servers.

### Feed Info

null

### Reproduction Video

https://github.com/user-attachments/assets/dbfc51a3-f691-4a3a-9d03-f02f41897627

### Environment

**App Version**: 0.1.2-beta.0
**OS**: Windows
**User Agent**: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Follow/0.1.2-beta.0 Chrome/130.0.6723.44 Electron/33.0.0 Safari/537.36
**Env**: electron
**Browser**: Chrome

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.

### Contributions

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests (actually just go ahead and do it, thanks!)

**Comment by EnterMan123** [comment]
I agree. As a heavy user, it's becoming increasingly difficult to tolerate the constant refreshing whenever I switch between feeds, categories, or views. I have to watch this zooming animation and the spinning loading icon hundreds of times a day, which seriously affects the overall user experience.

Sometimes it takes 5 to 15 seconds to load, this subtle delay subtly altering your usage habits, as if you must wait until it's finished before performing any other actions.

https://github.com/user-attachments/assets/55e9cf88-5b24-4cc4-9464-b99dbcdaff73

**Comment by Innei** [comment]
Try to fix in #1355

**Comment by EnterMan123** [comment]
https://github.com/user-attachments/assets/cf9edf60-a873-46da-8f6c-d7e663346881

闪现
![Image](https://github.com/user-attachments/assets/ce445dd9-83a1-400b-87a1-c5425eca490c)
